products/sle15/profiles/pci-dss.profile

```diff
@@ -68,11 +72,17 @@ selections:
     - audit_rules_usergroup_modification_passwd
     - audit_rules_usergroup_modification_shadow
     - audit_rules_sysadmin_actions
+    - dconf_gnome_screensaver_idle_activation_enabled
     - display_login_attempts
     - file_groupowner_grub2_cfg
     - file_owner_grub2_cfg
     - gid_passwd_group_same
+    - install_hids
     - no_empty_passwords
+<<<<<<< HEAD
```

Comment:
there are some conflicts here. I think it should be easier if you start from scratch on this one.